charts/px-backup/templates/px-central-ui/pxcentral-ui-configmap.yaml

```diff
@@ -1,6 +1,5 @@
 {{/* Setting defaults if they are omitted. */}}
```

Comment:
do we need this configmap if we are reading everything from the secrets?

Reply:
Removed